### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by PangMo5**
# Changes
- Add Weblate related contents
- Add Build Process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +22,27 @@
 
 <a href='https://testflight.apple.com/join/WiN0G62Q'><img height='70' alt='Join the Beta on TestFlight' src='https://anotherlens.app/testflight-badge.png'/></a>
 
+**Don't see Swiftfin in your language?**
```
Comment (acvigue) [inline README.md:25]: SwiftFin, capital F

**Review by acvigue — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (acvigue) [reply README.md:25]: That's all I can see. Looks good otherwise :)

Comment (PangMo5) [reply README.md]: fixed XD

**Comment by acvigue** [comment]
pressed wrong button at first lol

**Pull request merged**